lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1013,11 +1013,14 @@ class TestVideoDescriptorStudentViewJson(TestCase):
         'file_size': 222,
     }
     TEST_EDX_VIDEO_ID = 'test_edx_video_id'
+    TEST_YOUTUBE_ID = 'test_youtube_id'
+    TEST_YOUTUBE_EXPECTED_URL = 'http://youtu.be/test_youtube_id'
 
     def setUp(self):
         super(TestVideoDescriptorStudentViewJson, self).setUp()
+        video_declaration = "<video display_name='Test Video' youtube_id_1_0=\'" + self.TEST_YOUTUBE_ID + "\'>"
         sample_xml = (
-            "<video display_name='Test Video'> " +
+            video_declaration +
             "<source src='" + self.TEST_SOURCE_URL + "'/> " +
             "<transcript language='" + self.TEST_LANGUAGE + "' src='german_translation.srt' /> " +
             "</video>"
```

Comment:
Is `''.join([first, second, third])` preferable to `first.join([second, third])`?